Mathlib/CategoryTheory/Core.lean

```diff
@@ -40,8 +40,8 @@ instance coreCategory : Groupoid.{v₁} (Core C) where
 
 namespace Core
 
-@[simp]
 /- Porting note: abomination -/
+@[simp]
```

Comment:
This is moving the comment above the open Classical, to make the linter in #22670 happy. (If this should be a doc comment instead, I'm happy to change that aspect instead.)
(Besides, I'm not sure if the porting note still tracks anything useful, but that's for another PR.)